syntax = "proto2";

//protoc adc_reading.proto --nanopb_out=./

message AdcReadings{  
	required uint32 reading1 = 1;  
	required uint32 reading2 = 2;  
	required uint32 reading3 = 3;  
	required uint32 reading4 = 4;  
}
